doc/specs/Keyboard-Selection.md

```diff
@@ -58,21 +61,48 @@ Every combination of the `Direction` and `SelectionExpansionMode` will map to a
 **NOTE**: If `copyOnSelect` is enabled, we need to make sure we update the clipboard on every change in selection.
 
 ### Mark Mode
-Upon mark mode being enabled by a user keybinding (`ToggleMarkMode`), the current cursor position becomes the selection anchor. The arrow keys move the selection anchor by cell. Using the arrow keys while holding `Shift` moves the second selection anchor by cell. Additionally, using the arrow keys while holding `Ctrl` and `Shift` moves the second selection anchor by word. This allows the user to make a selection of their choice. The user can exit mark mode by using the `ToggleMarkMode` keybinding.
+Mark Mode is a mode where the user can create more complete and complex selections using only the keyboard. It can be cycled int an enabled/disabled state through a user keybinding (`toggleMarkMode`).
+
+Upon being enabled, the current cursor position becomes the selection anchor. The user can move the selection anchor using the same keybindings defined for keyboard selection (`moveSelectionAnchor` action).
+
+#### `toggleMarkMode`
+`toggleMarkMode` has two keybinding arguments:
+| Parameter | Accepted Values | Default Value | Description |
+|--|--|--|--|
+| `anchorKeybinding` | regular keybindings and plain modifier keys | `Shift` | The keybinding used to anchor the selection anchor to it's current text buffer position. While holding down that keybinding, any `moveSelectionAnchor` actions will move the other selection anchor (`endSelectionPosition`). |
+| `anchorMode` | `Hold`, `Toggle` | `Hold` | If `Hold`, the user needs to hold the `anchorKeybinding` keybinding to move the second selection anchor. If `Toggle`, the user switches between selection anchors when they press it. |
+
+#### Rendering during Mark Mode
+Since we are just moving the selection anchors, rendering the selection rects should operate normally. We need to ensure that we still scroll when we move a selection anchor past the top of bottom of the viewport.
+
+In ConHost, output would be paused when a selection was present. This is a completely separate issue that is being tracked in (#2529)[https://github.com/microsoft/terminal/pull/2529].
+
+#### Exiting Mark Mode
+There are multiple ways to exit mark mode. The user can press...
+- the `toggleMarkMode` keybinding
+- the `ESC` key
+- the `copy` keybinding (which also copies the contents of the selection to the user's clipboard)
+
+
+
+<!--Holding down the `holdSelectionAnchor` keybinding action (set to `Shift` by default) anchors the selection anchor to it's current text buffer position. While holding down that keybinding, any `moveSelectionAnchor` actions will move the other selection anchor (`endSelectionPosition`).
+
+**NOTE**: `holdSelectionAnchor` will have an `Enum anchorSwitching` parameter. Where `Hold` (default) is the functionality mentioned above, and `Toggle` switches between the selection anchors when pressed.-->

```

Comment:
I don't think I understand what's happening here

Reply:
Dead comment. Removed.

Also, mark mode discussion just got moved to #5804 to make our lives easier.